src/go/configgenerator/listener_generator.go

```diff
@@ -315,6 +317,11 @@ func makePathMatcherFilter(serviceInfo *sc.ServiceInfo) *hcmpb.HttpFilter {
 func makeGrpcStatsFilter() *hcmpb.HttpFilter {
 	cfg := &gspb.FilterConfig{
 		EmitFilterState: true,
+		PerMethodStatSpecifier: &gspb.FilterConfig_StatsForAllMethods{
+			StatsForAllMethods: &wrappers.BoolValue{
+				Value: true,
```

Comment:
Do we need this?

Reply:
This feature is used to make statistics of grpc methods and details are [here](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/grpc_stats/v3/config.proto#envoy-v3-api-msg-extensions-filters-http-grpc-stats-v3-filterconfig).

Setting it as true is same with our existing default config. In term of security, maybe we add a flag to toggle this feature and disable it by default.